docs/scenes/get-started/set-up-your-environment/instructions/_xcodeInstructions.mdx

```diff
@@ -22,3 +22,11 @@ Open Xcode, choose **Settings...** from the Xcode menu (or press <kbd>cmd ⌘</k
 />
 
 </Step>
+
+<Step label="3">
+
+### Install Watchman
+
+Watchman is a tool by Facebook for watching changes in the filesystem. It is highly recommended you install it for better performance. See [installation instructions](https://facebook.github.io/watchman/docs/install#macos) in Watchman documentation.
```

Comment:
```suggestion
[Watchman](https://facebook.github.io/watchman/docs/install#macos) is a tool for watching changes in the filesystem. Installing it will result in better performance. You can install it with:
```

Then, can we inline the command to install it instead of sending to other docs?

```shell
brew update
brew install watchman
```

Reply:
Done:

![Uploading CleanShot 2024-05-29 at 15.10.05@2x.png…]()